content/docs/contributing/release-process.md

```diff
@@ -487,43 +507,40 @@ page if a step is missing or if it is outdated.
 
-11. Check that the build is complete and send Slack messages about the release:
-
-    1. For recent versions of cert-manager, the build will have been automatically
-       triggered by the tag being pushed earlier. You can check if it's complete on
-       the [GCB Build History](https://console.cloud.google.com/cloud-build/builds?project=cert-manager-release).
-
-    2. If you're releasing an older version of cert-manager (earlier than 1.10) then the automatic build
-       will failed because the GCB config for that build wasn't backported.
-       In this case, you'll need to trigger the build manually using `cmrel`, which takes about 5 minutes:
-
-        ```bash
-        # Must be run from the "cert-manager/release" repo folder.
-        cmrel makestage --ref=$RELEASE_VERSION
-        ```
-
-        This build takes ~5 minutes. It will build all container images and create
-        all the manifest files, sign Helm charts and upload everything to a storage
-        bucket on Google Cloud. These artifacts will then be published and released
-        in the next steps.
+11. Check that the build that was automatically triggered when you pushed the
+    tag is complete and send Slack messages about the release:
 
-    3. In any case, send a first Slack message to `#cert-manager-dev`:
+    1. Send a first Slack message to `#cert-manager-dev`:
 
         <div className="pageinfo pageinfo-primary"><p>
         Releasing <code>1.2.0-alpha.2</code> 🧵
         </p></div>
 
-        <div className="pageinfo pageinfo-info"><p>
-        🔰 Please have a quick look at the build log as it might contain some unredacted
-        data that we forgot to hide. We try to make sure the sensitive data is
-        properly redacted but sometimes we forget to update this.
-        </p></div>
-
-    4. Send a second Slack message in reply to this first message with the
-       Cloud Build job link. For example, the message might look like:
+    2. Check that the build completed in the
+       [GCB Build History](https://console.cloud.google.com/cloud-build/builds?project=cert-manager-release).
+
+          <div className="pageinfo pageinfo-info"><p>
+          🔰 Please have a quick look at the build log as it might contain some unredacted
+          data that we forgot to hide. We try to make sure the sensitive data is
+          properly redacted but sometimes we forget to update this.
+          </p></div>
+
+       > **NOTE (1.10 and earlier):** If you're releasing an older version of
+       > cert-manager then the automatic build will failed because the GCB
+       > config for that build wasn't backported. In this case, you'll need to
+       > trigger the build manually using `cmrel`, which takes about 5 minutes:
+       >
+       > ```bash
+       > # Must be run from the "cert-manager/release" repo folder.
+       > cmrel makestage --ref=$RELEASE_VERSION
+       > ```
```

Comment:
Could we just remove this section? I once accidentally ran this command because I was confused.

Reply:
I made the same mistake, that's why I quoted it. I propose to have this PR merged and then let's create another one to remove this useless block.

Reply:
I would prefer we keep this information even though we don't support 1.9 and below. I have "gated" the information behind the text "**(version 1.10 and above)**", would that suffice to prevent confusion?